DscResources/MSFT_RegistryResource/MSFT_RegistryResource.psm1

```diff
@@ -332,6 +363,20 @@ function Set-TargetResource
                     $null = Remove-Item -Path $Key -Recurse -Force
                 }
             }
+            else
+            {
+                if ($Exclusive)
```

Comment:
Add a comment about why are removing:
* key exist
* value name not specified
* Exclusive specified

So remove any values.